packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/AccordianText.css

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright (c) 2017 Uber Technologies, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.AccordianText--header {
+  cursor: pointer;
+  overflow: hidden;
+  padding: 0.25em 0.1em;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.AccordianText--header:hover {
+  background: #e8e8e8;
+}
+
+.AccordianText--header.is-empty {
+  background: none;
+  cursor: initial;
+}
+
+.AccordianText--header.is-high-contrast:not(.is-empty):hover {
+  background: #ddd;
+}
+
+.AccordianText--emptyIcon {
+  color: #aaa;
+}
```

Comment:
I don't think these styles are used anywhere?

- `.AccordianText--header.is-empty`
- `.AccordianText--header.is-high-contrast:not(.is-empty):hover` (as a result of the above)
- `.AccordianText--emptyIcon`